src/components/datepicker/datePicker.js

```diff
@@ -330,7 +330,7 @@
     if (this.$attrs['ngDisabled']) {
       // The expression is to be evaluated against the directive element's scope and not
       // the directive's isolate scope.
-      var scope = this.$mdUtil.validateScope(this.$element) ? this.$element.scope() : null;
+      var scope = this.$scope.$parent;
```

Comment:
`$mdUtil.validateScope` is only used by `datePicker.js` and `util.js` 
And in util.js, it is only used to scan for focusable.

Anyway to fix the util.js or **remove** that fragile utility?